### PR TITLE
Replace SIG/WG meeting placeholder references with Zoom links-3

### LINKED ISSUE #4588: Replace SIG/WG meeting placeholder references with Zoom links
Some time ago Zoom did not have sufficient moderation capabilities and the Zoom meeting links were removed from [sigs.yaml] and pointed towards this placeholder:
https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit

As a workaround to help mitigate trolls from causing problems in meetings. This has since been resolved and the entries in [sigs.yaml] should be updated with their correct link.

There are 53 references to the above placeholder as of 2020-03-09. Updated zoom links can be taken from either the SIG/WG's agenda or their calendar event from the [community calendar](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America/Los_Angeles).

/area community-management
/sig contributor-experience
/help
/good-first-issue

[sigs.yaml]: https://github.com/kubernetes/community/blob/master/sigs.yaml

### PR DESCRIPTION
**Pull request by AGMETEOR**
**Which issue(s) this PR fixes**:
Fixes #4588

### CODE REVIEW

**Comment by AGMETEOR** [comment]
Some notes
-------
* Azure Subproject Meeting (Third Tuesday UTC, Monday Pacific)- Zoom link for time “1:00am UTC/6:00pm PST” not available

* Regular GCP Subproject Meeting - Zoom link for time “4pm UTC/9:00am PST” not available

* Regular IBM Subproject Meeting - Zoom link for time “2pm EST/11am PST” not available

File: sigs.yaml [hunk 1]
```diff
@@ -761,15 +761,15 @@ sigs:
...
+      url: https://zoom.us/j/586836662
       archive_url: https://docs.google.com/document/d/1SpxvmOgHDhnA72Z0lbhBffrfe9inQxZkU9xqlafOW9k/edit
       recordings_url: https://www.youtube.com/watch?v=yQLeUKi_dwg&list=PL69nYSiGNLP2JNdHwB8GxRs2mikK7zyc4
     - description: Azure Subproject Meeting (Third Tuesday UTC, Monday Pacific)
       day: Tuesday
       time: "1:00"
       tz: UTC
       frequency: monthly
-      url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
+      url: no-zoom-link
```
Comment (mrbobbytables) [inline sigs.yaml]:
```suggestion
      url: https://zoom.us/j/586836662
```
It looks like they use the same zoom link for both 👍

**Pull request merged**